### DISCUSSION TITLE
Armenian Language

### DISCUSSION THREAD

**Discussion by vahanhovsepyan** [monkeytypegame/monkeytype#5388] [Ideas]
I suggest adding Armenian language.
I see there is western Armenian, however we use Armenian (otherwise Eastern Armenian) in modern Armenia.
Not a programmer, but I'm ready to assist with whatever I can.

**Comment by Whydidthechickencrosstheroad** [comment]
Do you speak Armenian or is there someone you know who speaks Armenian? Are you talking about Armenian, eastern Armenian or both?

**Reply by vahanhovsepyan** [reply]
I am Armenian, and I do speak Armenian. Armenian and Eastern Armenian are the same.

Nowadays, in Armenia people speak in Eastern Armenian and we call it Armenian. While Western Armenian is only used by Armenians in Turkey, in Syria, in Lebanon, in France, in USA, in Europe who flew from ancient Armenia (modern eastern Turkey) during 1910-1930 period to survive from the Armenian Genocide.

I see we have Western Armenian in the system only, and we don't have nowadays Armenian, that's why I suggest to add it, since it is used in more than 98% of Armenian writings online.

**Reply by Whydidthechickencrosstheroad** [reply]
They added Armenian already. It is called Armenian and Armenian 1k

**Reply by Whydidthechickencrosstheroad** [reply]
Hello, I already told you it was added

**Comment by Whydidthechickencrosstheroad** [comment]
This is already been added

**Comment by Whydidthechickencrosstheroad** [comment]
Can someone mark this it is already answered and close this
